lib/Slackbot_worker.js

```diff
@@ -27,11 +27,11 @@ module.exports = function(botkit, config) {
             botkit.debug('CANNOT SEND WEBHOOK!!');
 
             return cb && cb('No webhook url specified');
-        }
+  }
 
         request.post(bot.config.incoming_webhook.url, function(err, res, body) {
-            if (err) {
-                botkit.debug('WEBHOOK ERROR',err);
+          if (err) {
```

Comment:
I am not familiar with it adding pretest command. You sound you know what you're doing :+1: 

Can you add it to package.json with this pull request? Thank you.